_episodes/05-func.md

```diff
@@ -346,20 +366,97 @@ DATA centered around the value.
 
 > ## Testing a Function
 >
-> 1. Write a function called `rescale` that takes an array as input and returns an
+> 1. Write a function called `normalise` that takes an array as input and returns an
```

Comment:
I realize that `normalise` is the more technically correct term here, but do you think it may be confusing to learners without a statistics background?

Reply:
I'm not sure it's a problem - the next line explains what is meant i.e. 
> with values scaled ....